buzzmobile/launch/simulation.launch

```diff
@@ -0,0 +1,30 @@
+<launch>
+    <arg name="mission_control" default="false"/>
+    <arg name="gzclient" default="false" />
+    <arg name="world" default="$(find buzzmobile)/simulation/models/world/empty.world"/>
+
+    <group ns="buzzmobile">
+        <!-- include file="$(find buzzmobile)/launch/includes/params.launch"/ -->
+        <!-- include file="$(find buzzmobile)/launch/includes/gps.launch"/ -->
+        <!-- include file="$(find buzzmobile)/launch/includes/mapping.launch"/ -->
+        <!-- include file="$(find buzzmobile)/launch/includes/steering.launch"/ -->
+        <!-- include file="$(find buzzmobile)/launch/includes/lidar.launch"/ -->
+        <!-- include file="$(find buzzmobile)/launch/includes/controller.launch"/ -->
+        <!-- group if="$(arg mission_control)" -->
+            <!-- include file="$(find buzzmobile)/launch/includes/visualization.launch"/ -->
+        <!-- /group -->
+        <include file="$(find buzzmobile)/launch/includes/sim_interface.launch"/>
+    </group>
+
+    <!-- Start gazebo server, load world, optionally start viz client -->
+    <node pkg="gazebo_ros" name="gazebo" type="gzserver"
+          args="-e ode $(arg world) use_sim_time:=true"
+          output="screen"/>
+    <group if="$(arg gzclient)">
+        <node pkg="gazebo_ros" name="gazebo_gui" type="gzclient"/>
+    </group>
+
+    <!-- Spawn car -->
+    <!-- node pkg="buzzmobile" name="sim_spawn_car" type="sim_spawn_car.sh"/ -->
```

Comment:
since this is working without the bash script, can we delete that and close #158 ?